code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -531,6 +531,33 @@
 	tastes = list("paint thinner" = 1)
 	color = "#EE35FF"
 
+/obj/item/reagent_containers/food/snacks/chewable/bubblegum/bubblegum
+	name = "bubblegum gum"
+	desc = "A rubbery strip of gum. You don't feel like eating it is a good idea."
+	color = "#F0B3B3"
+	list_reagents = list(/datum/reagent/blood = 15)
+	tastes = list("hell" = 1)
+
+/obj/item/reagent_containers/food/snacks/chewable/bubblegum/bubblegum/process()
+	..()
+	if(iscarbon(loc))
+		bbgumhal(loc)
+
+
+/obj/item/reagent_containers/food/snacks/chewable/bubblegum/bubblegum/On_Consume(mob/living/eater)
+	. = ..()
+	if(iscarbon(eater))
+		bbgumhal(eater)
```

Comment:
what is this proc name it reads like im having a stroke

Reply:
bubblegum hallucination

Reply:
Proc names should hint to their use, please dont shorten this.

Reply:
fixed